api/res/openapi.yml

```diff
@@ -185,6 +207,29 @@ paths:
             text/plain:
               schema:
                 type: string
+  /tasks/{taskId}:
```

Comment:
As stated below, to have a better naming `/apps/{appName}/status-changes/{statusId}` would be my choice for naming this resource. In this PR we can ignore `appName` in the route implementation:

```rust
#[get("/<app_name>/status-changes/<status_change_id>", format = "application/json")]
fn task(
    _app_name: Result<AppName, AppNameError>,
    status_change_id: Result<AppStatusChangeId, AppStatusChangeIdError>,
    apps: State<Arc<Apps>>,
    tasks: State<Tasks>,
    options: RunOptions,
) -> Result<AsyncCompletion<Json<Vec<Service>>>, HttpApiProblem> {
   // …
```

I'll create than an issue that we will use the `app_name` and remove the deployment guard.